newIDE/app/src/ProjectsStorage/CloudStorageProvider/CloudProjectWriter.js

```diff
@@ -243,7 +251,7 @@ export const generateOnSaveProjectAs = (
   }
   options.onStartSaving();
 
-  const gameId = saveAsLocation.gameId || project.getProjectUuid();
```

Comment:
why this change? was this not used?

Reply:
I saw it used only once, above in `generateOnChooseSaveProjectAsLocation`. I felt like it was not onChooseSaveProjectAsLocation role to set the gameId. So I removed it.